pallets/issuance/src/lib.rs

```diff
@@ -58,11 +58,18 @@ pub struct TgeInfo<A> {
 pub trait PoolPromoteApi {
```

Comment:
it doesnt seem to be pool promote api anymore, please rename it properly and provide doc for the function